### PR TITLE
Update the file_path attribute in Parser class to actual path in host

### PR DESCRIPTION
**Pull request by chenlizhong**
- previous file_path is the file path -- archive extract directory path
- Update failed tests

### CODE REVIEW

**Comment by chenlizhong** [comment]
Parser is logic layer. It do not need to care about extract file path. So update the path to relative_path.

File: insights/core/__init__.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ class Parser(object):
     """
 
     def __init__(self, context):
-        self.file_path = context.path
+        self.file_path = os.path.join("/", context.relative_path) if context.relative_path is not None else None
```
Comment (xiangce) [inline insights/core/__init__.py]: Should we just add a `self.relative_path` to `Parser` and leave the `self.file_path` as it was?

**Comment by csams** [comment]
Should we have the / prefix on the relative paths?

File: insights/core/__init__.py (hunk 1 above)
Comment (chenlizhong) [reply insights/core/__init__.py]: Originally, we think keep these two. But when I do the update. I think file_path and relative_path that will confuse others that consume the parser. In fact, the file_path in context is just exact temp dir, to evaluate one same archive two times, the value of file_path will be different also. So do not keep the value.  just keep relative_path as path here.

The parser

Comment (xiangce) [reply insights/core/__init__.py]: Yep, currently, I also cannot imagine what the path that contains the temp dir can be used for yet. That makes sense to me.

**Pull request merged**